.mage/js.go

```diff
@@ -169,7 +168,7 @@ func (js Js) Deps() error {
 
 // Build runs all necessary commands to build the console bundles and files.
 func (js Js) Build() {
-	mg.SerialDeps(js.Deps, JsSDK.Build, js.BuildDll, js.BuildMain)
+	mg.SerialDeps(js.Deps, js.BuildDll, js.BuildMain)
```

Comment:
This will be changed in https://github.com/TheThingsNetwork/lorawan-stack/pull/2751